requirements.txt

```diff
@@ -1,5 +1,5 @@
 Django==2.2.28
```

Comment:
Hi @marcu,
sorry for the dumb question, but shouldn't the django version also be updated?

Reply:
My idea was to prepare the migration to Django==3.2.

With the libraries updates I proposed, g3w-adlmin runs on Django==2 & on Django 3